plugins/org.polarsys.capella.scenario.editor.dsl.ui/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ui/TextualScenarioUiModule.java

```diff
@@ -1,15 +1,3 @@
-/*******************************************************************************
- * Copyright (c) 2020 THALES GLOBAL SERVICES.
- *  
- *  This program and the accompanying materials are made available under the
- *  terms of the Eclipse Public License 2.0 which is available at
- *  http://www.eclipse.org/legal/epl-2.0
- *  
- *  SPDX-License-Identifier: EPL-2.0
- *  
- *  Contributors:
- *     Thales - initial API and implementation
- ******************************************************************************/
 /**

```

Comment:
No copyright.

Reply:
There were 2 copyright headers, I removed one of them.